src/utils/marketsAndNetworksConfig.ts

```diff
@@ -183,7 +177,7 @@ export const getProvider = (chainId: ChainId): ethersProviders.Provider => {
 export const getENSProvider = () => {
   const chainId = 1;
   const config = getNetworkConfig(chainId);
-  return new ethersProviders.StaticJsonRpcProvider(config.publicJsonRPCUrl[0], chainId);
+  return new StaticJsonRpcProvider(config.publicJsonRPCUrl[0], chainId);
 };
```

Comment:
Do we want/need a unit test for the `getProvider` function, since it actually is implementing our rotation provider?

Reply:
Yeah good idea, added 2 new unit tests around this